Makefile

```diff
@@ -463,6 +463,12 @@ release-only:
 		echo 'Please update Added: tags in the documentation first.' ; \
 		exit 1 ; \
 	fi
+	@if $(grep -IPqrs '^>>>>>>> [0-9A-Fa-f]+' benchmark deps doc lib src test tools) || \
```

Comment:
Right.  We'd need three checks if we wanted to be really thorough but the third one isn't practical.  It's not perfect but two checks is still better than one.